docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,187 +1,523 @@
-settings specified in the stream's matching
-<<create-index-template,index template>>.
+* Watcher
++
+[source,console]
+----
+POST _watcher/_start
+----
+--
 
-If you change index settings during a restore, we recommend you make similar
-changes in the stream's matching index template. This ensures new backing
-indices created for the stream use the same index settings.
-====
-// end::index-settings-data-stream-warning[]
+. If wanted, reset the `action.destructive_requires_name` cluster setting.
++
+[source,console]
+----
+PUT _cluster/settings
+{
+  "persistent": {
+    "action.destructive_requires_name": null
+  }
+}
+----
+
+[discrete]
+[[monitor-restore]]
+=== Monitor a restore
+
+The restore operation uses the <<indices-recovery,shard recovery process>> to
+restore an index's primary shards from a snapshot. While the restore operation
+recovers primary shards, the cluster will have a `yellow`
+<<cluster-health,health status>>.
+
+After all primary shards are recovered, the replication process creates and
+distributes replicas across eligible data nodes. When replication is complete,
+the cluster health status typically becomes `green`.
+
+You can monitor the cluster health status using the <<cluster-health,cluster
+health API>>.
+
+[source,console]
+----
+GET _cluster/health
+----
+
+To get detailed information about ongoing shard recoveries, use the
+<<indices-recovery,index recovery API>>.
+
+[source,console]
+----
+GET my-index/_recovery
+----
+// TEST[setup:setup-snapshots]
+
+To view any unassigned shards, use the <<cat-shards,cat shards API>>.
+
+[source,console]
+----
+GET _cat/shards?v=true&h=index,shard,prirep,state,node,unassigned.reason&s=state
+----
+
+Unassigned shards have a `state` of `UNASSIGNED`. The `prirep` value is `p` for
+primary shards and `r` for replicas. The `unassigned.reason` describes why the
+shard remains unassigned.
+
+To get a more in-depth explanation of an unassigned shard's allocation status,
+use the <<cluster-allocation-explain,cluster allocation explanation API>>.
+
+[source,console]
+----
+GET _cluster/allocation/explain?filter_path=index,node_allocation_decisions.node_name,node_allocation_decisions.deciders.*
+{
+  "index": "my-index",
+  "shard": 0,
+  "primary": false,
+  "current_node": "my-node"
+}
+----
+// TEST[s/^/PUT my-index\n/]
+// TEST[s/"primary": false,/"primary": false/]
+// TEST[s/"current_node": "my-node"//]
 
 [discrete]
-=== Restoring to a different cluster
+[[cancel-restore]]
+=== Cancel a restore
 
-The information stored in a snapshot is not tied to a particular cluster or a cluster name. Therefore, it's possible to
-restore a snapshot created from one cluster into another cluster by registering the repository that contains the snapshot in the new cluster and starting the restore process.
+You can delete an index or data stream to cancel its ongoing restore. This also
+deletes any existing data for the index or data stream.
```

Comment:
Suggest clarifying somehow that we mean the existing data _in the cluster_, the data in the snapshot is unaffected.